ThunderCloud/LocalisedString.swift

```diff
@@ -44,6 +44,9 @@ public extension String {
     func localised(with key: String, paramDictionary: [String: Any]?) -> String {
         var string = NSAttributedString(string: self).localised(with: key, paramDictionary: paramDictionary).string
         string.localisationKey = key
+        if LocalisationsTool.showDebugLocalisations && !string.hasPrefix("[\(key)]") {
```

Comment:
@simonmitchell This code will get hit A LOT and I'm a little worried about this constantly checking the `UserDefaults`. I _thiiiiink_ the `UserDefaults` is `Dictionary` in memory which updates a file when mutated. But if there are any IO (or DB) operations at all we need to cache this flag in memory ourselves.

Reply:
I'll add in-memory caching, that's a great shout!